solutions/0100-0199/0199-binary-tree-right-side-view-medium.md

```diff
@@ -87,4 +87,76 @@ class Solution {
 }
 ```
 </TabItem>
+
+
+<TabItem value="py" label="Python">
+<SolutionAuthor name="@kondekarshubham123"/>
+
+```py
+
+"""
+
+binary-tree-right-side-view
+
+    3       <-  3
+   / \
+  9   20    <-  20
+     /  \
+    15   7  <-  7
+
+Output: [3, 20, 7]
+"""
+from __future__ import annotations
+from typing import List, Optional
```

Comment:
what is the reason to include these?

Reply:
Those imports are for annotations purpose only

Reply:
removing these lines